Composer/packages/client/src/telemetry/AppInsightsClient.ts

```diff
@@ -29,7 +30,7 @@ export default class AppInsightsClient {
   }
 
   public static trackEvent(name: string, properties: LogData) {
-    if (this._telemetrySettings?.allowDataCollection) {
+    if (alwaysTrackEvents.includes(name as TelemetryEventName) || this._telemetrySettings?.allowDataCollection) {
```

Comment:
I think this should be a separate condition that just logs the event straight to the server. There might be other events in the queue that we want to log.